packages/react-static/src/static/__mocks__/defaultConfigProduction.mock.js

```diff
@@ -1,5 +1,5 @@
 export default {
-  siteRoot: '',
+  siteRoot: '/',
```

Comment:
Relates to https://github.com/react-static/react-static/pull/1272/files#diff-fceff08b7598386eba9929c07035d93dR133

Reply:
I think this is fine?